src/orquestra/sdk/_base/_dsl.py

```diff
@@ -1151,6 +1156,11 @@ def task(
             result of other task) - it will be placeholded. Every character that is
             non-alphanumeric will be changed to dash ("-").
             Also only first 128 characters of the name will be used
+        retries: Maximum number of times a worker will try to retry after failure.
+            Useful if worker is killed by random events, or memory leaks from previously
+            executed tasks.
+            WARNING: retried workers might cause issues in MLFlow logging, as retried
+            workers are sharing invocationId, MLFlow tag will be shared across them.
```

Comment:
**nitpick*: `MLflow` vs `MLFlow`, tags vs "identifiers" ([tag in MLflow](https://mlflow.org/docs/latest/tracking/tracking-api.html#add-tags-to-runs) is a specific thing, while this issue can happen for any identifier)

```suggestion
            WARNING: retried workers might cause issues in MLflow logging, as retried
            workers share the same invocation ID, MLflow identifier will be shared across them.
```